internal/handlers/common/commands.go

```diff
@@ -0,0 +1,152 @@
+		Handler: (Handler).MsgBuildInfo,
+	},
+	"collStats": {
+		Help:    "Storage data for a collection.",
+		Handler: (Handler).MsgCollStats,
+	},
+	"create": {
+		Help:    "Creates the collection.",
+		Handler: (Handler).MsgCreate,
+	},
+	"dataSize": {
+		Help:    "Returns the size of the collection in bytes.",
+		Handler: (Handler).MsgDataSize,
+	},
+	"dbStats": {
+		Help:    "Returns the statistics of the database.",
+		Handler: (Handler).MsgDBStats,
+	},
+	"drop": {
+		Help:    "Drops the collection.",
+		Handler: (Handler).MsgDrop,
+	},
+	"dropDatabase": {
+		Help:    "Deletes the database.",
+		Handler: (Handler).MsgDropDatabase,
+	},
+	"getCmdLineOpts": {
+		Help:    "Returns a summary of all runtime and configuration options.",
+		Handler: (Handler).MsgGetCmdLineOpts,
+	},
+	"getLog": {
+		Help:    "Returns the most recent logged events from memory.",
+		Handler: (Handler).MsgGetLog,
+	},
+	"getParameter": {
+		Help:    "Returns the value of the parameter.",
+		Handler: (Handler).MsgGetParameter,
+	},
+	"hostInfo": {
+		Help:    "Returns a summary of the system information.",
+		Handler: (Handler).MsgHostInfo,
+	},
+	"ismaster": {
+		Help:    "Returns the role of the FerretDB instance.",
+		Handler: (Handler).MsgIsMaster,
+	},
+	"isMaster": { // both `ismaster` and `isMaster` are valid
+		Help:    "Returns the role of the FerretDB instance.",
+		Handler: (Handler).MsgIsMaster,
+	},
+	"hello": {
+		Help:    "Returns the role of the FerretDB instance.",
+		Handler: (Handler).MsgHello,
+	},
+	"listCollections": {
+		Help:    "Returns the information of the collections and views in the database.",
+		Handler: (Handler).MsgListCollections,
+	},
+	"listDatabases": {
+		Help:    "Returns a summary of all the databases.",
+		Handler: (Handler).MsgListDatabases,
+	},
+	"ping": {
+		Help:    "Returns a pong response. Used for testing purposes.",
+		Handler: (Handler).MsgPing,
+	},
+	"serverStatus": {
+		Help:    "Returns an overview of the databases state.",
+		Handler: (Handler).MsgServerStatus,
+	},
+	"whatsmyuri": {
+		Help:    "An internal command.",
+		Handler: (Handler).MsgWhatsMyURI,
+	},
+
+	"count": {
+		Help:    "Returns the count of documents that's matched by the query.",
+		Handler: (Handler).MsgCount,
+	},
+	"createIndexes": {
+		Help:    "Creates indexes on a collection.",
+		Handler: (Handler).MsgCreateIndexes,
+	},
+	"delete": {
+		Help:    "Deletes documents matched by the query.",
+		Handler: (Handler).MsgDelete,
+	},
+	"find": {
+		Help:    "Returns documents matched by the query.",
+		Handler: (Handler).MsgFind,
+	},
+	"findAndModify": {
+		Help:    "Inserts, updates, or deletes, and returns a document matched by the query.",
+		Handler: (Handler).MsgFindAndModify,
+	},
+	"insert": {
+		Help:    "Inserts documents into the database.",
+		Handler: (Handler).MsgInsert,
+	},
+	"update": {
+		Help:    "Updates documents that are matched by the query.",
+		Handler: (Handler).MsgUpdate,
+	},
+
+	// internal commands
```

Comment:
They are not "internal" anymore, no need to separate them with a blank line and comment